packages/visualizations/src/components/Table/EmptyRow.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+    export let label = 'No records found...';
+</script>
+
+<tr>
+    <div>
+        <em>{label}</em>
+    </div>
+</tr>
```

Comment:
`div` can't be inside a `tr`. I think it's incorrect for HTML.

Reply:
Indeed, you're right! Fixed in https://github.com/opendatasoft/ods-dataviz-sdk/pull/258/commits/e9dacf7d3cd574e9d0d8fde0661f8f1e03ee8d36

I struggled a bit to do what I wanted with a `td` that why I used a `div`.
But everything looks fine with a `td`, I guessed my CSS wasn't good enough at first.